examples/pcrglobwb.ipynb

```diff
@@ -2,28 +2,28 @@
  "cells": [
```

Comment:
<p>I am getting</p><pre class="ql-syntax" spellcheck="false">/home/verhoes/git/eWaterCycle/ewatercycle/ewatercycle/models/pcrglobwb.py:48: UserWarning: No valid input path received for temperatureNC, defaulting to ''.
</pre><p> </p><p>I worked around it by doing </p><p> </p><pre class="ql-syntax" spellcheck="false">cp examples/pcrglobwb_example_case/forcing/temperature_2001to2010.nc&nbsp;&nbsp;examples/
</pre><p> </p><p>Can this be added as shell command or as instructions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
This is an advanced use cases that I decided to remove from the notebook.